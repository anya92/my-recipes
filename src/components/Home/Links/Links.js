import React from 'react';
import styled from 'styled-components';
import { Link } from 'react-router-dom';

const StyledLinks = styled.div`
  max-width: 100%;
  display: flex;
  justify-content: center;
  flex-wrap: wrap;
  a {
    font-size: 1.2rem;
    color: #333;
    text-decoration: none;
    margin: 10px 30px;
  }
`;

const Links = () => (
  <StyledLinks>
    <Link to="/breakfast">Breakfast</Link>
    <Link to="/dinner">Dinner</Link>
    <Link to="/desserts">Desserts</Link>
    <Link to="/snacks">Snacks</Link>
    <Link to="/drinks">Drinks</Link>
  </StyledLinks>
);

export default Links;
