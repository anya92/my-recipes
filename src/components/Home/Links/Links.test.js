import React from 'react';
import { shallow } from 'enzyme';

import Links from './Links';

describe('Links', () => {
  test('should render Links correctly', () => {
    const wrapper = shallow(<Links />);
    expect(wrapper).toMatchSnapshot();
  });
});
