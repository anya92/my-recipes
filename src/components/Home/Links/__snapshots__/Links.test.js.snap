// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Links should render Links correctly 1`] = `
<styled.div>
  <Link
    replace={false}
    to="/breakfast"
  >
    Breakfast
  </Link>
  <Link
    replace={false}
    to="/dinner"
  >
    Dinner
  </Link>
  <Link
    replace={false}
    to="/desserts"
  >
    Desserts
  </Link>
  <Link
    replace={false}
    to="/snacks"
  >
    Snacks
  </Link>
  <Link
    replace={false}
    to="/drinks"
  >
    Drinks
  </Link>
</styled.div>
`;
